packages/blockchain-wallet-v4-frontend/src/data/components/signMessage/sagas.js

```diff
@@ -8,19 +8,18 @@ import { promptForSecondPassword } from 'services/SagaService'
 import { Types } from 'blockchain-wallet-v4/src'
 import * as signer from 'blockchain-wallet-v4/src/signer'
 
+const taskToPromise = t => new Promise((resolve, reject) => t.fork(reject, resolve))
```

Comment:
This is a great helper function that seems to be defined in a few places. Is there a module that this can live in?